public/app/features/geo/utils/frameVectorSource.ts

```diff
@@ -43,6 +43,7 @@ export class FrameVectorSource<T extends Geometry = Geometry> extends VectorSour
       return;
     }
 
+    //eslint-disable-next-line
```

Comment:
@nmarrs -- I think this is a reasonable fix for now.  I think this whole function will get replaced soon :)